### ISSUE TITLE
IMOS: check_longitude_variable - check_latitude_variable - variable not float or double

### ISSUE THREAD

**Issue by lbesnard** [aodn/compliance-checker#37]
netcdf checker complains:

```
var                                    :3:   124/142 :  
    LATITUDE                           :3:    15/18 :  
        check_variable_type            :3:     0/ 1 : The Type of variable
                                                      LATITUDE is not Double or
                                                      Float
...
    LONGITUDE                          :3:    15/18 :  
        check_variable_type            :3:     0/ 1 : The Type of variable
                                                      LONGITUDE is not Double or
                                                      Float

```

However, ncdump :

```
        float LATITUDE(TIME) ;
                LATITUDE:long_name = "latitude" ;
...
        float LONGITUDE(TIME) ;
                LONGITUDE:long_name = "longitude" ;
```

There's obviously a bug somewhere

file 
SOOP/SOOP-SST/9V2768_Wakmatha/2015/IMOS_SOOP-SST_T_20150528T000000Z_9V2768_FV01_C-20150624T041657Z.nc

**Comment by fxmzb123** [comment]
@lbesnard @mhidas  this has been fixed and in pull request #42

**Comment by lbesnard** [comment]
thx
